File: ParadigmasdaProgramacao/atividadeJava/src/questoes_java/Aluno.java
package questoes_java;

import java.util.ArrayList;

public class Aluno {
    private String nomeAluno;
    private Float primeiraNota;
    private Float segundaNota;
    private Float terceiraNota;
    private Float notaFinal;
    private String Resultado;

    public Aluno(){
    }

    public Float calcularMediaAluno (Float primeiraNota, Float segundaNota, Float terceiraNota){
        this.notaFinal = (primeiraNota + segundaNota + terceiraNota) ;
        return this.notaFinal;
    }

    public String getNomeAluno() {
        return nomeAluno;
    }

    public void setNomeAluno(String nomeAluno) {
        this.nomeAluno = nomeAluno;
    }

    public Float getPrimeiraNota() {
        return primeiraNota;
    }

    public void setPrimeiraNota(Float primeiraNota) {
        this.primeiraNota = primeiraNota;
    }

    public Float getSegundaNota() {
        return segundaNota;
    }

    public void setSegundaNota(Float segundaNota) {
        this.segundaNota = segundaNota;
    }

    public Float getTerceiraNota() {
        return terceiraNota;
    }

    public void setTerceiraNota(Float terceiraNota) {
        this.terceiraNota = terceiraNota;
    }

    public Float getNotaFinal() {
        return notaFinal;
    }

    public void setNotaFinal(Float notaFinal) {
        this.notaFinal = notaFinal;
    }

    public String getResultado() {
        return Resultado;
    }

    public void setResultado(String resultado) {
        Resultado = resultado;
    }
}

File: ParadigmasdaProgramacao/atividadeC#/Aluno.cs
using System;

public class Aluno {
    
    private String nomeAluno;
    
    private double primeiraNota;
    
    private double segundaNota;
    
    private double terceiraNota;
    
    private double notaFinal;
    
    private String Resultado;
    
    public Aluno() {
    }
    
    public double CalcularMediaAluno(double primeiraNota, double segundaNota, double terceiraNota) {
        this.notaFinal = (this.primeiraNota + (this.segundaNota + this.terceiraNota));
        return this.notaFinal;
    }
    
    public String GetNomeAluno() {
        return this.nomeAluno;
    }
    
    public void SetNomeAluno(String nomeAluno) {
        this.nomeAluno = nomeAluno;
    }
    
    public double GetPrimeiraNota() {
        return this.primeiraNota;
    }
    
    public void SetPrimeiraNota(double primeiraNota) {
        this.primeiraNota = primeiraNota;
    }
    
    public double GetSegundaNota() {
        return this.segundaNota;
    }
    
    public void SetSegundaNota(double segundaNota) {
        this.segundaNota = segundaNota;
    }
    
    public double GetTerceiraNota() {
        return this.terceiraNota;
    }
    
    public void SetTerceiraNota(double terceiraNota) {
        this.terceiraNota = terceiraNota;
    }
    
    public double GetNotaFinal() {
        return this.notaFinal;
    }
    
    public void SetNotaFinal(double notaFinal) {
        this.notaFinal = notaFinal;
    }
    
    public String GetResultado() {
        return this.Resultado;
    }
    
    public void SetResultado(String resultado) {
        this.Resultado = resultado;
        
    }
}